index.html

```diff
@@ -32,96 +32,100 @@ <h3> I'm a software developer </h3>
+        <li>Ruby</li>
+      </ul>
     </div>
-    <img src="img/Placeholder.svg" alt="">
-    <h2> Multi-Post Stories</h2>
-    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-  <div class="box">
-    <button type="button"> CSS</button>
-    <button type="button">html</button>
-    <button type="button">Bootstrap</button>
-    <button type="button">Ruby</button>
-  </div>
-  <button type="button"> See Project </button>
-  <form class="segment"> 
-    <div class="container">
-    <h1> Professional Art Printing Data </h1> 
-    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-    <div class="holder">  
-    <div>html</div>
-    <div>Bootstrap</div>
-    <div>Ruby</div>
-  </div>
-</div>
-    <button type="button" class="see-project"> See Project </button>
-  </form>
-  <form class="segment"> 
-    <div class="container">
-    <h1> Professional Art Printing Data </h1> 
-    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-    <div class="holder">  
-    <div>html</div>
-    <div>Bootstrap</div>
-    <div>Ruby</div>
-  </div>
-</div>
-    <button type="button" class="see-project"> See Project </button>
-  </form> <form class="segment"> 
-    <div class="container">
-    <h1> Professional Art Printing Data </h1> 
-    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-    <div class="holder">  
-    <div>html</div>
-    <div>Bootstrap</div>
-    <div>Ruby</div>
-  </div>
-</div>
-    <button type="button" class="see-project"> See Project </button>
-  </form>
-  <form class="segment"> 
-    <div class="container">
-    <h1> Professional Art Printing Data </h1> 
-    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-    <div class="holder">  
-    <div>html</div>
-    <div>Bootstrap</div>
-    <div>Ruby</div>
-  </div>
-</div>
-    <button type="button" class="see-project"> See Project </button>
-  </form>
-  <form class="segment"> 
+    <div class="see-project-button">
+      <button type="submit">See Project</button>
+    </div>
+    </div>
+  </section>
+  <section class="box">
+    <form>
     <div class="container">
-    <h1> Professional Art Printing Data </h1> 
-    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-    <div class="holder">  
-    <div>html</div>
-    <div>Bootstrap</div>
-    <div>Ruby</div>
-  </div>
-</div>
+      <h1> Professional Art Printing Data </h1> 
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
```

Comment:
- [ ] You did an amazing job but kindly following semantic HTML you are not allowed to use the h1 tag more than once. So please change it. 💪